NOTICE

```diff
@@ -0,0 +1,5 @@
+Change(s)
```

Comment:
Not sure if this is needed

Reply:
I think lets just add it

Reply:
@mabdh This notice can not be added globally. Only new files can be added a new license or copyright.

Reply:
@haveiss lets remove this file mas?